packages/plugin-css/test/fixtures/type-transition/want.css

```diff
@@ -17,142 +17,142 @@
   --ds-transition-cubic-out-quick-timing-function: var(--ds-ease-cubic-out);
   --ds-transition-cubic-out-moderate: var(--ds-transition-cubic-out-moderate-duration) var(--ds-transition-cubic-out-moderate-delay) var(--ds-transition-cubic-out-moderate-timing-function);
   --ds-transition-cubic-out-moderate-duration: var(--ds-timing-moderate);
-  --ds-transition-cubic-out-moderate-delay: 0ms;
+  --ds-transition-cubic-out-moderate-delay: 0;
   --ds-transition-cubic-out-moderate-timing-function: var(--ds-ease-cubic-out);
   --ds-transition-cubic-out-deliberate: var(--ds-transition-cubic-out-deliberate-duration) var(--ds-transition-cubic-out-deliberate-delay) var(--ds-transition-cubic-out-deliberate-timing-function);
   --ds-transition-cubic-out-deliberate-duration: var(--ds-timing-deliberate);
-  --ds-transition-cubic-out-deliberate-delay: 0ms;
+  --ds-transition-cubic-out-deliberate-delay: 0;
   --ds-transition-cubic-out-deliberate-timing-function: var(--ds-ease-cubic-out);
   --ds-transition-cubic-out-slow: var(--ds-transition-cubic-out-slow-duration) var(--ds-transition-cubic-out-slow-delay) var(--ds-transition-cubic-out-slow-timing-function);
   --ds-transition-cubic-out-slow-duration: var(--ds-timing-slow);
-  --ds-transition-cubic-out-slow-delay: 0ms;
+  --ds-transition-cubic-out-slow-delay: 0;
   --ds-transition-cubic-out-slow-timing-function: var(--ds-ease-cubic-out);
   --ds-transition-linear-instant: var(--ds-transition-linear-instant-duration) var(--ds-transition-linear-instant-delay) var(--ds-transition-linear-instant-timing-function);
   --ds-transition-linear-instant-duration: var(--ds-timing-instant);
-  --ds-transition-linear-instant-delay: 0ms;
+  --ds-transition-linear-instant-delay: 0;
   --ds-transition-linear-instant-timing-function: var(--ds-ease-linear);
   --ds-transition-linear-quick: var(--ds-transition-linear-quick-duration) var(--ds-transition-linear-quick-delay) var(--ds-transition-linear-quick-timing-function);
   --ds-transition-linear-quick-duration: var(--ds-timing-quick);
-  --ds-transition-linear-quick-delay: 0ms;
+  --ds-transition-linear-quick-delay: 0;
   --ds-transition-linear-quick-timing-function: var(--ds-ease-linear);
   --ds-transition-linear-moderate: var(--ds-transition-linear-moderate-duration) var(--ds-transition-linear-moderate-delay) var(--ds-transition-linear-moderate-timing-function);
   --ds-transition-linear-moderate-duration: var(--ds-timing-moderate);
-  --ds-transition-linear-moderate-delay: 0ms;
+  --ds-transition-linear-moderate-delay: 0;
   --ds-transition-linear-moderate-timing-function: var(--ds-ease-linear);
   --ds-transition-linear-deliberate: var(--ds-transition-linear-deliberate-duration) var(--ds-transition-linear-deliberate-delay) var(--ds-transition-linear-deliberate-timing-function);
   --ds-transition-linear-deliberate-duration: var(--ds-timing-deliberate);
-  --ds-transition-linear-deliberate-delay: 0ms;
+  --ds-transition-linear-deliberate-delay: 0;
```

Comment:
Hm this seems to be a regression but is probably before your PR since the tests were disabled. We can fix before shipping another release though

Reply:
This is probably because of this check in `transformDurationValue`: https://github.com/Design-System-Pro/terrazzo/blob/6a3346cc2931637e2ee9a2e0d54d58e9842a9cd5/packages/token-tools/src/css/duration.ts#L14

I actually thought was by design since this also happens in the `transformDimensionValue`: 
https://github.com/Design-System-Pro/terrazzo/blob/6a3346cc2931637e2ee9a2e0d54d58e9842a9cd5/packages/token-tools/src/css/dimension.ts#L14-L15

Reply:
Yeah you’re right. I think it was just a bad copy-paste. For some reason CSS requires units for durations always, even though dimensions may be unitless.